### PR TITLE
Fix failing build

### PR DESCRIPTION
**Pull request by SirDomin**
Added conflict to api-platform 2.6.
Removed ecs due to BC break.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -43,7 +43,6 @@
         "phpstan/phpstan": "0.12.29",
         "phpstan/phpstan-webmozart-assert": "0.12.6",
         "phpunit/phpunit": "^8.5",
-        "sylius-labs/coding-standard": "^3.2",
```
Comment (pamil) [inline composer.json:46]: Let's leave it there

File: composer.json [hunk 2]
```diff
@@ -63,13 +62,13 @@
...
     "conflict": {
-        "symplify/package-builder": "^8.3.25"
+        "symplify/package-builder": "8.3.25",
+        "api-platform/core": "^2.6"
     },
     "scripts": {
         "analyse": [
             "@composer validate --strict",
-            "vendor/bin/phpstan.phar analyse -c phpstan.neon -l max src/",
-            "vendor/bin/ecs check src/ spec/"
```
Comment (pamil) [inline composer.json:72]: And just remove this line

File: composer.json (hunk 1 above)
Comment (pamil) [reply composer.json:46]: It's not preventing us from installing anything, it's a runtime error

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, @SirDomin! :1st_place_medal:

**Pull request merged**